provider/cmd/pulumi-resource-azuread/schema.json

```diff
@@ -8735,181 +8735,180 @@
                         },
-                        "description": "A collection of `required_resource_access` blocks as documented below.\n"
+                        "type": "array"
                     },
                     "serviceManagementReference": {
-                        "type": "string",
-                        "description": "References application context information from a Service or Asset Management database.\n"
+                        "description": "References application context information from a Service or Asset Management database.\n",
+                        "type": "string"
                     },
                     "signInAudience": {
-                        "type": "string",
-                        "description": "The Microsoft account types that are supported for the current application. One of `AzureADMyOrg`, `AzureADMultipleOrgs`, `AzureADandPersonalMicrosoftAccount` or `PersonalMicrosoftAccount`.\n"
+                        "description": "The Microsoft account types that are supported for the current application. One of `AzureADMyOrg`, `AzureADMultipleOrgs`, `AzureADandPersonalMicrosoftAccount` or `PersonalMicrosoftAccount`.\n",
+                        "type": "string"
                     },
                     "singlePageApplications": {
-                        "type": "array",
+                        "description": "A `single_page_application` block as documented below.\n",
                         "items": {
                             "$ref": "#/types/azuread:index/getApplicationSinglePageApplication:getApplicationSinglePageApplication"
                         },
-                        "description": "A `single_page_application` block as documented below.\n"
+                        "type": "array"
                     },
                     "supportUrl": {
-                        "type": "string",
-                        "description": "URL of the application's support page.\n"
+                        "description": "URL of the application's support page.\n",
+                        "type": "string"
                     },
                     "tags": {
-                        "type": "array",
+                        "description": "A list of tags applied to the application.\n",
                         "items": {
                             "type": "string"
                         },
-                        "description": "A list of tags applied to the application.\n"
+                        "type": "array"
                     },
                     "termsOfServiceUrl": {
-                        "type": "string",
-                        "description": "URL of the application's terms of service statement.\n"
+                        "description": "URL of the application's terms of service statement.\n",
+                        "type": "string"
                     },
                     "webs": {
-                        "type": "array",
```

Comment:
Do you know what happened to all of these? I don't see anything in your changes which would cause this.

Reply:
These aren't semantic changes. IDK, something changed in Go struct devs for JSON serde? 

```
                     "users": {
-                        "type": "array",
+                        "description": "A list of users. Each `user` object provides the attributes documented below.\n",
                         "items": {
                             "$ref": "#/types/azuread:index/getUsersUser:getUsersUser"
                         },
-                        "description": "A list of users. Each `user` object provides the attributes documented below.\n"
+                        "type": "array"
                     }
```